### PR TITLE
ci: Enable a naive CI for GstVkVideoParsers

### PR DESCRIPTION
**Pull request by dabrain34**

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dependecies
+      run: |
+        sudo apt update
+        sudo apt install pkg-config
+        sudo apt install libglib2.0-dev libjson-glib-dev libncursesw5-dev
+        sudo apt install python3-pip python3-setuptools ninja-build
+        sudo apt install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev
+        sudo pip3 install --user meson
```
Comment (ceyusa) [inline .github/workflows/pipeline.yml:24]: I think there's no need for meson from pip... perhaps just from apt ... we'll  need to check

File: .github/workflows/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Dependecies
+      run: |
+        sudo apt update
+        sudo apt install pkg-config
+        sudo apt install libglib2.0-dev libjson-glib-dev libncursesw5-dev
```
Comment (ceyusa) [inline .github/workflows/pipeline.yml:21]: afaik, there's no need for these manually installed dependencies

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (dabrain34) [reply .github/workflows/pipeline.yml:24]: I always prefer the pip3 one as it's more up2date. The ubuntu 22.04 version is 0.61, pip3 is 0.63.2 and minimum gstreamer version is 0.62

File: .github/workflows/pipeline.yml (hunk 2 above)
Comment (dabrain34) [reply .github/workflows/pipeline.yml:21]: removed

**Pull request closed without merging**